report-pull-request-results.py

```diff
@@ -174,12 +179,9 @@ def read_addon_log_file(unit_tests_file):
       tname, err = cmd_to_addon_test(line.split(': FAILED -')[0].strip(), addon_dir)
       if not tname: tname = "unknown"
       else: tname = "[%s](%s/addOnTests/%s)" % (tname, options.report_url, tname)
-      if cnt<=max_show:
+      if cnt <= max_show:
         if err: line = err
-        if len(line.split('\n'))>1:
-          message += "- "+ tname + '\n```\n' + line + '\n```\n'
-        else:
-          message += "- "+ tname + '```' + line + '```\n'
+        message += "- "+ tname + '\n```\n' + line + '\n```\n'
```

Comment:
I think we could avoid the distinction between single-line and multi-line message. At the very least, the old single-line case is missing a whitespace between `tname` and " ```line " (old L182).